### PR TITLE
Refactor locals

### PR DESCRIPTION
**Pull request by bschaatsbergen**
## what
* Renamed locals for consistency.
* Added labels (had to introduce the google-beta provider in order to place the labels on each instance in the managed instance group).

## why
* Having more labels available helps with identifying resources in Google Cloud (billing, etc).
* The locals could use some love and consistency.

**Note**: this is likely a breaking change as we're adding labels - causing the instance template to be recreated.

### CODE REVIEW

**Comment by bschaatsbergen** [comment]
@kpocius, @drieske, @mvanholsteijn, @mmourick, @kvanzuijlen, @cblkwell would any of you like to review this change please?

**Review by bschaatsbergen — COMMENTED** [review]
TODO: test this change

File: README.md [hunk 1]
```diff
@@ -200,14 +202,14 @@ You can check the status of the certificate in the Google Cloud Console.
 
 | Name | Type |
 |------|------|
+| [google-beta_google_compute_instance_group_manager.default](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/google_compute_instance_group_manager) | resource |
```
Comment (kpocius) [inline README.md:205]:
```suggestion
| [google-beta_google_compute_instance_group_manager.default](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/compute_instance_group_manager) | resource |
```

Comment (bschaatsbergen) [reply README.md:205]: Hmm note @kpocius, the docs are auto-generated using `terraform-docs` - is there a particular reason you're suggesting this change?

Comment (bschaatsbergen) [reply README.md:205]: Interesting that it does this, your suggestion seems right.

Comment (kpocius) [reply README.md:205]: The link doesn't work `¯\_(ツ)_/¯` so I update it to the right one.

File: main.tf [hunk 2]
```diff
@@ -1,11 +1,16 @@
 locals {
-  # The default port that Atlantis runs on is 4141.
+  # The default port that Atlantis runs on is 4141, we default to this.
   atlantis_port = lookup(var.env_vars, "ATLANTIS_PORT", 4141)
-  # Atlantis its home directory is "/home/atlantis".
-  atlantis_data_dir    = lookup(var.env_vars, "ATLANTIS_DATA_DIR", "/home/atlantis")
-  port_name            = "atlantis"
-  network_traffic_tags = ["atlantis-${random_string.random.result}"]
-  labels               = merge(var.labels, { "container-vm" = module.container.vm_container_label })
+  # Atlantis its home directory is "/home/atlantis", we default to this.
```
Comment (kvanzuijlen) [inline main.tf:4]:
```suggestion
  # Atlantis' home directory is "/home/atlantis", we default to this.
```

**Review by cblkwell — APPROVED** [review]

**Comment by bschaatsbergen** [comment]
I'm moving these commits under #123 to release 1 major upgrade

**Pull request closed without merging**